### PR TITLE
Fixing capitalization and a typo

### PR DESCRIPTION
**Pull request by rajbos**
1. Spotted a typo to fix
1. All errors should start with the same capitalization and most started with lowercase, so I changed all occurrences in this file

### CODE REVIEW

File: controllers/autoscaling.go [hunk 1]
```diff
@@ -221,7 +221,7 @@ func (r *HorizontalRunnerAutoscalerReconciler) suggestReplicasByQueuedAndInProgr
 	necessaryReplicas := queued + inProgress
 
 	r.Log.V(1).Info(
-		fmt.Sprintf("Suggested desired replicas of %d by TotalNumberOfQueuedAndInProgressWorkflowRuns", necessaryReplicas),
+		fmt.Sprintf("suggested desired replicas of %d by TotalNumberOfQueuedAndInProgressWorkflowRuns", necessaryReplicas),
```
Comment (mumoshu) [inline controllers/autoscaling.go:224]:
```suggestion
		fmt.Sprintf("Suggested desired replicas of %d by TotalNumberOfQueuedAndInProgressWorkflowRuns", necessaryReplicas),
```

This is an informational message and every log message should start capitalized so let me revert this

File: controllers/autoscaling.go [hunk 2]
```diff
@@ -363,7 +363,7 @@ func (r *HorizontalRunnerAutoscalerReconciler) suggestReplicasByPercentageRunner
 	//   the runnerdeployment controller is replacing RunnerReplicaSet for runner update.
 
 	r.Log.V(1).Info(
-		fmt.Sprintf("Suggested desired replicas of %d by PercentageRunnersBusy", desiredReplicas),
+		fmt.Sprintf("suggested desired replicas of %d by PercentageRunnersBusy", desiredReplicas),
```
Comment (mumoshu) [inline controllers/autoscaling.go:366]:
```suggestion
		fmt.Sprintf("Suggested desired replicas of %d by PercentageRunnersBusy", desiredReplicas),
```

ditto

**Review by mumoshu — APPROVED** [review]
@rajbos Thanks a lot for the fixes! A few comments, but this looks good otherwise. I'll merge this in a few days

**Comment by rajbos** [comment]
👍

**Pull request merged**